internal/notify/kafka.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package notify
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/moov-io/achgateway/internal/kafka"
+	"github.com/moov-io/achgateway/internal/service"
+	"github.com/moov-io/achgateway/pkg/compliance"
+	"github.com/moov-io/achgateway/pkg/models"
+	"github.com/moov-io/base/log"
+
+	"gocloud.dev/pubsub"
+)
+
+type Kafka struct {
+	publisher *pubsub.Topic
+	cfg       *service.KafkaConfig
+}
+
+func NewKafka(logger log.Logger, cfg *service.KafkaConfig) (*Kafka, error) {
+	publisher, err := kafka.OpenTopic(logger, cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Kafka{publisher: publisher, cfg: cfg}, nil
+}
+
+type event struct {
```

Comment:
Did we want to specify an event type? So multiple message types could be sent on the topic?

Reply:
Probably? Ideally, types everywhere, just in case.